docs/manual/java/guide/build/EclipseSbt.md

```diff
@@ -41,26 +41,26 @@ Follow these steps to integrate your project with Eclipse:
 # Create an External Tool Configuration
 
 1. From the Eclipse toolbar, click **External Tools** and select **External Tools Configurations**.
-    [[EclExternalConfig.png]] 
+    [[EclExternalConfig.png]]
 1. In the left pane, select **Program** and click **New Launch Configuration**.
-    [[EclNewLaunch.png]] 
+    [[EclNewLaunch.png]]
 1. Create your configuration as follows:
     1. Enter a name.
     1. On the **Main** tab in the **Location** field, enter the location of your sbt installation.
     For Windows, browse to the location where you installed sbt and select `sbt.bat`. On Linux, you can find the location of sbt by opening a terminal and entering `which sbt`.
     1. For **Working Directory** browse to the top-level sbt project folder. (This is the folder containing the `build.sbt` file.)
     1. In the **Arguments** field, enter `runAll`.
     1. Click **Apply**. Your screen should look similar to the following:
-    [[EclSbtConfig.png]] 
+    [[EclSbtConfig.png]]
 1. Click **Run**.
     On success, the console shows that the services are running.
     [[EclSbtSuccess.png]]
-    
-    Verify that the services are indeed up and running by invoking the `hello` service endpoint from any HTTP client, such as a browser: 
-        
-        ```
-        http://localhost:9000/api/hello/World
-        ```
```

Comment:
The reason the triple backticks appear in the rendered output here is that the code sample is indented an additional four spaces. GitHub-flavored Markdown offers two different ways to format code blocks: add an extra four-space indent, or use triple backticks before and after the block. Because this block did both, it interpreted the triple backticks as part of the block of code that should be escaped. Either removing the backticks or aligning them to the same column as the "V" in "Verify" above would have the intended result: render the code block without the backticks, but still as part of the list item.